src/api/goods.ts

```diff
@@ -1,11 +1,25 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+function wait(delay: number) {
+  return new Promise(resolve => setTimeout(resolve, delay));
+}
+
+export async function getAll(): Promise<Good[]> {
+  await wait(2000);
```

Comment:
2 seconds is waay tooo much, you can literally remove it, or make it as small as possible